llama_index/vector_stores/simple.py

```diff
@@ -136,8 +136,8 @@ def query(
         # TODO: consolidate with get_query_text_embedding_similarities
         items = self._data.embedding_dict.items()
 
-        if query.doc_ids:
-            available_ids = set(query.doc_ids)
+        if query.node_ids:
```

Comment:
unrelated but we should prob also change docstring in `SimpleVectorStoreData` to explicitly say node_id to embeddings instead of doc_id

Reply:
Yea good point, added that